1) dcolor.py

mrkraimer opened the pull request:
a) def __init__(self, samples=1000, xmin=-8, xmax=8, ymin=-8, ymax=8):
   samples 1000 instead of 3500 makes examples more responsive
b) added title and x and y label
c) removed all examples
d) toolbar no longer appears

2) dcolorExample.py
Many examples.
When each image appears clicking the close button starts the next example.
On linux the button appears on the upper right corner

3) rgbcolor.py
Alternative to dcolar.
I think it better shows the magnitude of abs(z).
white means big.
shades of green means intermediate.
black means small.

4) rgbcolorExample.py and  rgbcolorExampleExpand.py are for rgbcolor

5) .gitignore is new

mrkraimer committed:
1) dcolor.py
a) def __init__(self, samples=1000, xmin=-8, xmax=8, ymin=-8, ymax=8):
   samples 1000 instead of 3500 makes examples more responsive
b) added title and x and y label
c) removed all examples
d) toolbar no longer appears

2) dcolorExample.py
Many examples.
When each image appears clicking the close button starts the next example.
On linux the button appears on the upper right corner

3) rgbcolor.py
Alternative to dcolar.
I think it better shows the magnitude of abs(z).
white means big.
shades of green means intermediate.
black means small.

4) rgbcolorExample.py and  rgbcolorExampleExpand.py are for rgbcolor

5) .gitignore is new

mrkraimer committed:
1) dcolorExample.py added example
2) dcolorExample.py added example like dcolorExample.py; changed expand for last example

mrkraimer committed:
more changes to examples

hernanat commented:
hey @mrkraimer thanks for your changes, I've been on the road a lot the past few days so haven't had a chance to review. Will do it as soon as I can, glad you figured out what you needed to!

mrkraimer committed:
yet more changes to example

hernanat commented:
hey @mrkraimer I'm back in the swing of things, going to review this today. Thanks again!

mrkraimer committed:
added hsv

hernanat commented:
hey @mrkraimer this looks really good! Would you mind updating the README to account for your additions? Also what do you think about placing the examples under a `/examples` directory?

mrkraimer committed:
update README

mrkraimer committed:
update README

mrkraimer commented:
I updated README.md

mrkraimer committed:
add z**z to examples

mrkraimer committed:
make hsvcolor show images as dcolor

hernanat commented:
> Also what do you think about placing the examples under a `/examples` directory?

this comment / suggestion still stands -- or I can pull your branch and change it too.

mrkraimer committed:
move examples to subdirectory

mrkraimer commented:
I moved examples to sub-directory examples.
I also added examples/example.py.
It allows dcolorExample.py, hsvcolorExample.py, and rgbcolorExample.py to have the same set of dc.plot lamda expressions

hernanat commented:
good stuff @mrkraimer I'm going to make a couple of tweaks myself and then it'll be merged. Thanks for all the work you did!

mrkraimer commented:
We had a major storm last Tuesday and lost power and internet connection,
We got power back yesterday,
Today internet assess is again working.

mrkraimer committed:
minor changes to examples

hernanat commented on hsvcolor.py:
```diff
@@ -0,0 +1,121 @@
+        rgb = np.stack([r, g, b], axis=-1)
+
+        return rgb
+
```
extra newlines here, can we remove all but one?

hernanat reviewed (approved):
@mrkraimer one more minor comment but looks good. Can you do me a favor and squash your commits down into 1 and write a commit message as outlined in [this guide](https://thoughtbot.com/blog/5-useful-tips-for-a-better-commit-message)

mrkraimer commented:
I read
https://thoughtbot.com/blog/5-useful-tips-for-a-better-commit-message

But I do not understand how that  will squash my commits down into 1

hernanat commented:
> I read
> https://thoughtbot.com/blog/5-useful-tips-for-a-better-commit-message
> 
> But I do not understand how that will squash my commits down into 1

Hey @mrkraimer no worries, [this](https://www.git-tower.com/learn/git/faq/git-squash/) is a great guide on learning how to do that!

The thoughtbot guide is about writing good commit messages, squashing is separate. Sorry if that was confusing.

mrkraimer commented:
I read 

https://www.git-tower.com/learn/git/faq/git-squash/

I fear making a mess.
Can you squash  while you merge the pull request?

hernanat commented:
@mrkraimer I've merged your changes in https://github.com/hernanat/dcolor/commit/db16061917d0158bba4f393e78e06822cfe3607f.

Thanks again for all the work you put into this. I want to apologize for it taking this long for me to get your changes in, it's been really crazy on my end and I haven't had a lot of time to handle things outside of work.

mrkraimer commented:
Thanks for merging!!

I have merged my repository with yours.
I made no changes to your code.
I did notice one problem with the README.md

The hsvcolor image should be identical to the dcolor image.
